tests/e2e/serial/main_test.go

```diff
@@ -1520,6 +1520,96 @@ func TestSuspendScanSetting(t *testing.T) {
 	}
 }
 
+func TestRemoveProfileScan(t *testing.T) {
+	f := framework.Global
+
+	// Creates a new `ScanSetting`, where the actual scan schedule doesn't necessarily matter, but `suspend` is set to `False`
+	scanSettingName := framework.GetObjNameFromTest(t) + "-scansetting"
+	scanSetting := compv1alpha1.ScanSetting{
```

Comment:
We don't need a separate `ScanSetting` for this test, do we? Are we able to reuse an existing one without issues?